src/styles/content.scss

```diff
@@ -57,9 +60,177 @@ table.os-raise-standardtable {
   }
 }
 
+%os-raise-basictable {
+  /* border-collapse: separate paired with border-spacing: 0 allows for a border-radius
+     to be applied to the entire table (border-collapse: collapse will ignore any border
+     radius applied to the table)
+  */
+  border: 2px solid $raise-table-color;
+  border-collapse: separate;
+  border-spacing: 0;
+  border-radius: 5px;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  thead {
+    background-color: $raise-table-color;
+  }
+
+  th,
+  td {
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    font-size: 1.2rem;
+    font-weight: 700;
+    color: #fff;
+  }
+
+  td {
+    font-size: 1.067rem;
+    color: $raise-light-black;
+    background-color: #fff;
+    border-left: 1px solid $raise-table-color;
+    border-top: 1px solid $raise-table-color;
+  }
+}
+
+table.os-raise-doubleheadertable {
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  th,
+  td {
+    border: 1px solid $raise-light-gray;
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    color: #000;
+    font-size: 1.2rem;
+    background-color: #84ada8;
+    font-weight: 700;
+  }
+
+  thead tr th:first-child {
+    border: none;
+    background-color: #fff;
+  }
+
+  td {
+    color: $raise-light-black;
+    font-size: 1.067rem;
+    background-color: #fff;
+  }
+}
+
+table.os-raise-textheavytable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  th,
+  td {
+    text-align: left;
+  }
+
+  td:first-child {
+    border-left: none;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
+}
+
+table.os-raise-midsizetable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 66%;
+  margin: auto;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+table.os-raise-skinnytable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 240px;
+
+  td:first-child {
+    border-left: none;
+  }
+}
+
+.os-raise-skinnytablemedia {
+  margin: auto;
+}
```

Comment:
Was this class perhaps from testing / tinkering?  I see it's also used in the example HTML, but I wanted to confirm whether it was intentionally included. If so, maybe we should consider a more general utility class for allowing authors to apply `margin: auto` when desired :man_shrugging: ? If we go that route we should document the new utility class explicitly as well.

Reply:
I was not sure if it made more sense to make a general utility class or just keep it as a "companion" class. I am okay with making a general utility class. Since we are adjusting the margin to center the skinny tables, I also needed to be able to center the image/graph or else when the table wraps under the image/graph only the table was being centered.

Reply:
I'd lean towards defining / using a utility class in the example. It might come in handy for authors in other circumstances (e.g. to center a stand-alone image or something 🤷‍♂️ ). If you wanted to steal bootstrap naming conventions, it could be `os-raise-mx-auto`, but open to other suggestions / ideas.